SETUP.md

```diff
@@ -168,9 +168,9 @@ git push origin master
 You should be good to go now.
 A good first step is to modify `content/metadata.yaml` with the relevant information for your manuscript.
 
-# Merging upstream manubot-rootstock changes
+# Merging upstream rootstock changes
```

Comment:
Same question as above in this section.  What is the most intuitive way to refer to this repository?  By the repository name or the more descriptive name?

Reply:
I think we can just call it rootstock. I updated the title in https://github.com/greenelab/manubot-rootstock/pull/180/commits/2ac7411890e82cf831e258a89b86ca93676cb9e6 so it's a bit more clear.

Reply:
Your changes were good clarifications